python/lsst/validate/drp/base.py

```diff
@@ -17,9 +17,22 @@
 # You should have received a copy of the LSST License Statement and
 # the GNU General Public License along with this program.  If not,
 # see <https://www.lsstcorp.org/LegalNotices/>.
+"""lsst.validate.drp's Measurement API that handles JSON persistence.
+"""
 
```

Comment:
I would suggested separating this new stuff in `base.py` instead into `io.py` for the serialization stuff and perhaps a new `measurement.py` for the measurement objects.